### PR TITLE
adding custom action for setting cluster_environment

### PR DESCRIPTION
**Pull request by hsachan22**

### CODE REVIEW

File: actions/cluster-env-setter/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "cluster-env-setter",
```
Comment (ypmittal) [inline actions/cluster-env-setter/package.json:2]: please add the usage of this action inside our workflows within this PR

**Review by ypmittal — CHANGES_REQUESTED** [review]
address the 1 change requested

File: actions/cluster-env-setter/package.json (hunk 1 above)
Comment (hsachan22) [reply actions/cluster-env-setter/package.json:2]: Done

File: actions/cluster-env-setter/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  const dockerJsonMetadata = core.getInput('docker_json_metadata')
+  const dockerMetadata = JSON.parse(dockerJsonMetadata)
+
+  const imageVersion = dockerMetadata.labels['org.opencontainers.image.version']
+
+  if (!imageVersion || typeof imageVersion !== 'string') {
+    throw new Error('Unable to get image version from metadata')
+  }
+
+  const isTesting = imageVersion.match(/.*-rc[0-9]/)
```
Comment (ypmittal) [inline actions/cluster-env-setter/index.js:13]: default should be testing.
i.e. we should assign production namespace if and only if a production tag is pushed

**Review by ypmittal — APPROVED** [review]
LGTM

**Pull request merged**